torchtext/data/field.py

```diff
@@ -81,10 +81,9 @@ def preprocess(self, x):
         """Load a single example using this field, tokenizing if necessary."""
         if self.sequential and isinstance(x, six.string_types):
             x = self.tokenize(x)
+        x = Pipeline(six.text_type)(x)
```

Comment:
> have the torchtext public interface always be Unicode.

Not sure if i understood what you mean, doesn't this change make the torchtext public interface agnostic to `unicode` or `str` (since we just convert to `unicode` under the hood)?